src/applications/personalization/dashboard/tests/e2e/dashboard.minimal-widgets.cypress.spec.js

```diff
@@ -0,0 +1,91 @@
+import {
+  enrollmentStatusNotEnrolled,
+  makeUserObject,
+  mockLocalStorage,
+  userPreferencesEmpty,
+} from './dashboard-e2e-helpers';
+
+describe('MyVA Dashboard', () => {
+  describe('when user should see the minimal amount of widgets', () => {
+    beforeEach(() => {
```

Comment:
I don't think that we need the `beforeEach` here since there is only one test 🙂

Reply:
True. I kinda like it though since it makes it a little clearer that this is just setup code. Of course I could also wrap it in a `setup()` function and call that instead of using a `beforeEach`

Reply:
Nah, I'm not too opinionated. Looks good to me as is 👍